### DISCUSSION TITLE
The Play Idea List

### DISCUSSION THREAD

**Discussion by atapas** [reactplay/react-play#62] [Ideas]
#48 

Hello People,

I'm opening this discussion related to this issue # https://github.com/atapas/react-play/issues/48

Could you please suggest some play ideas you want to see or build in ReactPlay? We want to post those ideas as public for people to take inspiration from and build plays. Please provide details like play name, level, and details.

For example:

- Play name: Tic Tac Toe
- Level: Beginner
- Details: a game in which two players seek in alternate turns to complete a row, a column, or a diagonal with either three O's or three X's drawn in the spaces of a grid of nine squares; noughts and crosses. 

Please comment.

**Comment by atapas** [comment]
@atapas @nirmalkc @shrilakshmishastry @murtuzaalisurti @Abhishek-90 

Please feel free to pour in some ideas.

**Comment by atapas** [comment]
Check the reply to this one. 100+ Ideas

https://twitter.com/ReactPlayIO/status/1516285534884139015

**Comment by murtuzaalisurti** [comment]
Play name: Code Snippets

Level: Intermediate

Details: A simple web application which allows you to copy paste your code snippet and lets you to download it as an image to share!

**Reply by atapas** [reply]
@murtuzaalisurti How do you know that's my next open source project idea? 😀

It sounds great, + 1 from me to add that to the idea list.

**Comment by vasantisuthar** [comment]
Play name: Threads
Level: Intermediate
Details: Build a react application for creating threads like in Twitter with a model, allowing multiple threads, tweet size constraints, and display them on the home page.

**Comment by Abhishek-90** [comment]
Play Name: Animations
Level: Intermediate/Advance
Details: Application with various modern Animations that add an aesthetic feel to the website.

**Comment by atapas** [comment]
Team, the idea page UI is almost ready. We will publish close to 30 ideas soon.
